libs/portals/admin/signature-collection/src/screens-parliamentary/DownloadReports/MyPdfDocument/index.tsx

```
@@ -0,0 +1,111 @@
+import logo from './logo.png'
+import { dark200 } from '@island.is/island-ui/theme'
+import { SignatureCollectionAreaSummaryReport } from '@island.is/api/schema'
+
+const MyPdfDocument = ({
+  report,
+}: {
+  report: SignatureCollectionAreaSummaryReport
+}) => {
+  return (
+    <Document>
+      {report && report.lists?.length
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)